tests/storage/test_hotplug.py

```
@@ -213,44 +277,85 @@ def test_hotplug_volume_with_bus_and_persist_migrate(
+        hotplugged_dvs_scope_class: list[DataVolume],
         fedora_vm_for_hotplug_scope_class: VirtualMachineForTests,
     ):
-        if is_dv_migratable(dv=blank_disk_dv_multi_storage_scope_class):
+        if all(is_dv_migratable(dv=dv) for dv in hotplugged_dvs_scope_class):
             migrate_vm_and_verify(
                 vm=fedora_vm_for_hotplug_scope_class, client=admin_client, check_ssh_connectivity=True
             )
+        else:
+            LOGGER.warning(
+                f"Skipping migration for VM {fedora_vm_for_hotplug_scope_class.name}: "
+                "not all hotplugged DVs support RWX access mode"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)